web-rdfs/source/http_handlers.cc

```diff
@@ -11,11 +11,33 @@ void setZk(zkclient::ZkNnClient *zk_arg) {
 }
 
 void create_file_handler(std::shared_ptr<HttpsServer::Response> response,
-                         std::shared_ptr<HttpsServer::Request> request) {
+                         std::string path) {
   LOG(DEBUG) << "HTTP request: create_file_handler";
 
-  // TODO(security): implement
-  response->write("create_file_handler");
+  std::string input = "hdfs dfs -fs hdfs://localhost:5351 -put " + path;
+
+  hadoop::hdfs::CreateResponseProto res;
+  hadoop::hdfs::CreateRequestProto req;
+
+  req.set_src(path);
+
+  zkclient::ZkNnClient::CreateResponse zkResp = zk->create_file(req, res);
+
+  response->write(webRequestTranslator::getCreateResponse(path));
```

Comment:
since getCreateResponse writes the whole header use *response << webRequestTranslator::getCreateResponse(path)